public/index.css

```diff
@@ -0,0 +1,97 @@
+html, body {
+  height: 100%;
+}
+
+html {
+  display: table;
+  margin: 0 auto;
+}
+
+body {
+  display: table-cell;
+  font-family: 'Roboto', sans-serif;
+  vertical-align: middle;
+}
+
+.calculator-container {
+  background-color: rgb(156, 156, 156);
+  border-radius: 4px;
+  box-shadow: 0 10px 50px rgb(98, 98, 98);
+  height: 320px;
+  margin: 0 auto;
+  width: 232px;
+}
+
+/* calculation display div */
+.calculator-output {
+  box-sizing: border-box;
+  font-size: 2.8em;
+  font-weight: 200;
+  height: 79px;
+  padding-right: 15px;
+  padding-top: 30px;
+  text-align: right;
+}
+
+/* all buttons */
+.calculator-btn {
+  display: inline-block;
```

Comment:
this approach of floats is also old school. Its really great that you've gotten the experience using this old hack but in modern browsers flexbox would be a much cleaner way to achieve this layout. It would also solve the issue you have where your orange buttons are a few pixels too high.